modules/EmailTemplates/EmailTemplateParser.php

```diff
@@ -174,6 +174,18 @@ private function getValueFromBean($variable)
 
         $parts = explode($charUnderscore, ltrim($variable, $charVariable));
         list($moduleName, $attribute) = [array_shift($parts), join($charUnderscore, $parts)];
+
+        /* Leads/Prospects/Users have a special variable naming scheme.
```

Comment:
Removed, reverted, re-removed and now re-added again :laughing:. I'd potentially want to investigate this further to see exactly why this was done and what the differences are between this and the old parser but this looks good for now since you've added in unit tests.

Reply:
I don't understand it either why a new parser was added instead of fixing the old one. There could have been a good reason for it of course, not sure :)